agent/lm_agent/config.py

```diff
@@ -66,6 +66,12 @@ class Settings(BaseSettings):
     # path to the binary for olixtool (needed for OLicense licenses)
     OLIXTOOL_PATH: Path = _DEFAULT_BIN_PATH
 
+    # reservation name for reconciliation
+    RESERVATION_IDENTIFIER: str = "license-manager-reservation"
+
+    # License Manager user name to create reservations
+    LM_USER: str = "license-manager"
```

Comment:
I have a conceptual question about it: shouldn't the reservation be made on behalf of the user who requested the reservation, i.e. someone real?

Reply:
No, the reservation is made by license manager to block licenses that are already in use in other environments.
We'll provision an operator level user to run the agent and be able to manage the reservation.
This reservation is not to be used by any users, it's only to sync the counters for licenses in use.